### ISSUE TITLE
Support for running under browser profiles

### ISSUE THREAD

**Issue by honzajde** [simonw/shot-scraper#93]
This is especially useful to get rid of all those consent pop-ups that our modern legally compliant web is bloated with....

**Comment by simonw** [comment]
I'm not sure if this is possible using Playwright. This page of documentation looks like it might be relevant here? https://playwright.dev/docs/api/class-browsertype#browser-type-launch-persistent-context

**Comment by honzajde** [comment]
I have found several links, this one looks most relevant: 

https://github.com/microsoft/playwright/issues/13878

It looks like it is possible

On my machine profiles are here: `~/.config/google-chrome/<Default | Profile 1 | Profile 2 | ...>`
Profile of current window can be found via `chrome:version` as "Profile Path"

Futher, by reading this:  
https://github.com/microsoft/playwright/issues/13878#issuecomment-1116151751

It looks like: 

Profiles can not be shared with other windows! And they say that 
1. Playwright needs to run in it's own window 
2. Playwright can not connect to a running instance;) This was (is?) possible with puppeteer as far as I remember...

So in summary `launchPersistentContext ` should work under the above assumptions.
